pkg/appconsts/appconsts.go

```diff
@@ -55,22 +55,30 @@ const (
 	// in a continuation sparse share of a sequence.
 	ContinuationSparseShareContentSize = ShareSize - NamespaceSize - ShareInfoBytes
 
-	// DefaultMaxSquareSize is the maximum original square width.
+	// MaxSquareSize is the maximum original square width.
 	//
 	// Note: 128 shares in a row * 128 shares in a column * 512 bytes in a share
 	// = 8 MiB
-	DefaultMaxSquareSize = 128
+	MaxSquareSize = 128
+
+	// DefaultGovMaxSquareSize is the default value for the governance
+	// modifiable max square size.
+	DefaultGovMaxSquareSize = 64
+
+	// DefaultMaxBytes is the default value for the maximum number of bytes
+	// allowed in a valid block.
+	DefaultMaxBytes = DefaultGovMaxSquareSize * DefaultGovMaxSquareSize * ContinuationSparseShareContentSize
```

Comment:
> out of curiosity, why ContinuationSparseShareContentSize instead of ShareSize?

this is mainly just to not account for namespaces 

> However I don't expect it to be the true upper bound b/c we expect some compact shares for the PFB txs associated with the blobs.

I'm a bit confused by this, how are we not accounting for the PFBs here or what do compact shares have to do with this? max bytes convers BlobTxs, which contain the PFBs no? this is just to count the total number of bytes in a square, including compact shares, but not including any block encoding.

Reply:
> max bytes covers BlobTxs, which contain the PFBs no?

yep!

> what do compact shares have to do with this?

Sorry I should've clarified. `ContinuationSparseShareContentSize` is slightly larger than `ContinuationCompactShareContentSize`

> this is mainly just to not account for namespaces

Makes sense. I wanted to call out that `ContinuationSparseShareContentSize` won't account for the bytes needed for compact shares (i.e. reserved bytes) and since a full square with contain at least one compact share for the PFB tx and at least one starting sparse share, the upper bound is slightly too high. In other words

```
DefaultMaxBytes = DefaultGovMaxSquareSize * DefaultGovMaxSquareSize * ContinuationSparseShareContentSize
DefaultMaxBytes = 64 * 64 * 478
DefaultMaxBytes = 1,957,888
```

But a valid block can't have 1,957,888 bytes because that assumes the entire square is continuation blob shares. Therefore we could have a slightly lower upper bound. Alternatively can we use the more readable `ShareSize` because `MaxSquareSize` will get hit and take affect before `DefaultMaxBytes`?

Reply:
ahh ok I see, yeah this purely because it seemed like an easy way to get a more accurate estimate. We could switch to share size and it will not have an effect (especifally after https://github.com/celestiaorg/celestia-core/issues/1007 that is)

perhaps we should just use a round number then